src/ray/object_manager/object_manager_client_connection.cc

```diff
@@ -2,58 +2,24 @@
 
 namespace ray {
 
+uint64_t SenderConnection::id_counter_ = 0;
```

Comment:
static class variables should be initialized to zero implicitly. So this is unnecessary, I believe.